libfreerdp/core/server.c

```diff
@@ -687,30 +689,46 @@ HANDLE WINAPI FreeRDP_WTSOpenServerA(LPSTR pServerName)
 		return INVALID_HANDLE_VALUE;
 
 	vcm = (WTSVirtualChannelManager*) calloc(1, sizeof(WTSVirtualChannelManager));
+	if (!vcm)
+		return NULL;
```

Comment:
Shouldn't you return INVALID_HANDLE_VALUE here like in the other error cases below ?